### PR TITLE
[CCAP-204] Set up JobRunr in IL-GCC code

### PR DESCRIPTION
**Pull request by cram-cfa**
#### 🔗 Jira ticket
https://codeforamerica.atlassian.net/browse/CCAP-204

#### ✍️ Description
This PR gets JobRunr working and functional in the dev environment. Once the web application is running, there is a sample page at https://localhost:8080/jobs/sample-jobs

This is primarily based on https://www.jobrunr.io/en/documentation/configuration/spring/ and https://github.com/jobrunr/example-spring

We probably will only use a variation of the first, second, and (most likely) last links. Clicking on those will enqueue a very simple job. You'll be able to see the logging of the simple job in the console (as the job just spits out a println). 

You'll also be able to, in development, go to http://localhost:8000/ and see the JobRunr dashboard. That should show you the job status if you click around in the jobs section of the dashboard. This only appears to work on http, so you may need to hit this via Safari (I found that my Chrome, thanks to the work to enable https in dev, won't let me get to this dashboard now).

#### ✅ Completion tasks

- [ ] Added relevant tests
- [ ] Meets acceptance criteria

### CODE REVIEW

File: src/main/java/org/ilgcc/app/config/LocaleConfiguration.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import java.util.Locale;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
```
Comment (analoo) [inline src/main/java/org/ilgcc/app/config/LocaleConfiguration.java:5]: I looked a bit into ComponentScan and I don't think I understand how it is being used. Would love to talk about it!

**Review by analoo — APPROVED** [review]
This looks great. I tested and confirmed that I am able to enqueue different types of jobs. I added names to the parameter and then checked localhost:8000 and saw the jobs enqueued. 

I do feel like the fact that we can't open the JobRunr jobs dashboard in https seems like an issue but it has a workaround (using a safari) so this is not a blocking issue to me.

I have a question about Component Scan for my own curiosity but again this is not blocking either: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/annotation/ComponentScan.html

File: src/main/java/org/ilgcc/app/config/LocaleConfiguration.java (hunk 1 above)
Comment (coltborg) [reply src/main/java/org/ilgcc/app/config/LocaleConfiguration.java:5]: I don't think this resource is used in this file.

Comment (cram-cfa) [reply src/main/java/org/ilgcc/app/config/LocaleConfiguration.java:5]: Here's where I moved the thing I added! 

https://github.com/codeforamerica/il-gcc-form-flow/pull/337/commits/de14ad5ca5bb8dd0e37a03895e1a627b91164aff

But that change to IlGCCApplication is also gone now, so I wonder what changed.

File: src/main/java/org/ilgcc/app/IlGCCApplication.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 
-@SpringBootApplication(scanBasePackages = {"org.ilgcc.app", "formflow.library"})
+@SpringBootApplication(scanBasePackages = {"org.ilgcc.app", "formflow.library", "org.ilgcc.jobs"})
```
Comment (cram-cfa) [inline src/main/java/org/ilgcc/app/IlGCCApplication.java:7]: @analoo this is basically doing the same exact thing as a ComponentScan, and I realized it after originally using a ComponentScan annotation. I just then forgot to remove the import statement that caused you to wonder what it was doing!

File: Dockerfile [hunk 3]
```diff
@@ -8,6 +8,6 @@ WORKDIR /opt/il-gcc
 RUN ./gradlew assemble && \
     cp /opt/il-gcc/build/libs/app.jar app.jar
 
-EXPOSE 8080
+EXPOSE 8080 8000
```
Comment (cram-cfa) [inline Dockerfile:11]: @coltborg Hopefully this work!

File: src/main/java/org/ilgcc/app/config/LocaleConfiguration.java (hunk 1 above)
Comment (cram-cfa) [reply src/main/java/org/ilgcc/app/config/LocaleConfiguration.java]: Ok, this unused import is now removed!

**Review by coltborg — APPROVED** [review]
I think this looks good for getting started. Still somethings to iron out in the different environments, but we'll need to do testing in them.

**Pull request merged**